### PR TITLE
Assist: fix summary logic

### PR DESCRIPTION
**Pull request by hugoShaka**
Fixes a logic issue: we want to generate a summary if the buffer is not empty AND has not overflowed.

### CODE REVIEW

**Review by jakule — COMMENTED** [review]
Can you add a test to cover this?

**Comment by hugoShaka** [comment]
@jakule done

File: lib/web/command_utils_test.go [hunk 1]
```diff
@@ -140,3 +140,57 @@ func TestSummaryBuffer(t *testing.T) {
...
+
+func TestSummaryBuffer_CanBeExported(t *testing.T) {
+	type fields struct {
+		buffer  map[string][]byte
+		invalid bool
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   bool
```
Comment (jakule) [inline lib/web/command_utils_test.go:152]: nit: You can use `require.BoolAssertionFunc`

**Review by jakule — APPROVED** [review]

File: lib/web/command.go [hunk 2]
```diff
@@ -287,7 +287,8 @@ func (h *Handler) executeCommand(
 	runCommands(hosts, runCmd, h.log)
 
 	// Optionally, try to compute the command summary.
-	if output, overflow := buffer.Export(); !overflow || len(output) != 0 {
+	if buffer.CanBeExported() {
+		output, _ := buffer.Export()
```
[the remark this answers is not in the document]
Comment (hugoShaka) [reply lib/web/command.go]: 👍  I simplified the Export() signature and removed the check.

**Review by AntonAM — APPROVED** [review]

**Pull request merged**